webapp_tracet/trigger_app/signals.py

```diff
@@ -321,6 +323,28 @@ def send_all_alerts(trigger_bool, debug_bool, pending_bool, proposal_decision_mo
     # Make sure they are unique and put each on a new line
     telescopes = ", ".join(list(set(telescopes)))
 
+    # Work out when the source will go below the horizon
+    telescope = proposal_decision_model.proposal.telescope
+    location = EarthLocation(
+        lon=telescope.lon*u.deg,
+        lat=telescope.lat*u.deg,
+        height=telescope.height*u.m
+    )
+    obs_source = SkyCoord(
+        proposal_decision_model.ra,
+        proposal_decision_model.dec,
+        #equinox='J2000',
+        unit=(u.deg, u.deg)
+    )
+    # Convert from RA/Dec to Alt/Az
+    delta_24h = np.linspace(0, 1440, 288)*u.min # 24 hours in 5 min increments
+    next_24h = obstime=Time.now()+delta_24h
+    obs_source_altaz = obs_source.transform_to(AltAz(obstime=next_24h, location=location))
+    for altaz, time in zip(obs_source_altaz, next_24h):
+        if altaz.alt.deg <1.:
+            # source below horizon so record time
+            set_time_utc = time
```

Comment:
I think this will actually store the rise time:
- lopping over all alt/time
- store set time if alt <1
- set_time_utc will be updated for every occurance of alt<1
- the final value of set_time_utc will be the last time that alt<1 ie the rise time

solution:
```
if altaz.alt.deg <1.:
  set_time_utc = time
  break
```

Reply:
Fixed